HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -211,20 +211,21 @@ def self.apply_neighbor_buildings(hpxml)
     end
   end
 
-  def self.apply_building_occupancy(hpxml, nbeds)
+  def self.apply_building_occupancy(hpxml, nbeds, schedules_file)
     if hpxml.building_occupancy.number_of_residents.nil?
       hpxml.building_occupancy.number_of_residents = Geometry.get_occupancy_default_num(nbeds)
       hpxml.building_occupancy.number_of_residents_isdefaulted = true
     end
-    if hpxml.building_occupancy.weekday_fractions.nil? && hpxml.header.schedules_filepath.nil?
+    schedules_file_includes_occupants = Schedule.schedules_file_includes_col_name(schedules_file, Constants.Occupants)
+    if hpxml.building_occupancy.weekday_fractions.nil? && !schedules_file_includes_occupants
       hpxml.building_occupancy.weekday_fractions = Schedule.OccupantsWeekdayFractions
       hpxml.building_occupancy.weekday_fractions_isdefaulted = true
     end
-    if hpxml.building_occupancy.weekend_fractions.nil? && hpxml.header.schedules_filepath.nil?
+    if hpxml.building_occupancy.weekend_fractions.nil? && !schedules_file_includes_occupants
       hpxml.building_occupancy.weekend_fractions = Schedule.OccupantsWeekendFractions
       hpxml.building_occupancy.weekend_fractions_isdefaulted = true
     end
-    if hpxml.building_occupancy.monthly_multipliers.nil? && hpxml.header.schedules_filepath.nil?
+    if hpxml.building_occupancy.monthly_multipliers.nil? && !schedules_file_includes_occupants
```

Comment:
Aren't there a whole bunch more in this file that need to be updated?

Reply:
Oops. I think I forgot to finish the "first cut".

How do we handle interior lighting? It doesn't appear to get defaulted in this file.

Reply:
Looks like it pulls it from the epw. So, nothing to do I suppose.